client/core/core.go

```diff
@@ -422,12 +476,12 @@ func (c *Core) Register(form *Registration) (error, <-chan error) {
 
```

Comment:
The fee information is stored in the DB once the fee is paid, but you are right that there is not yet any attempt to recover in that scenario. Part of the issue is that upon starting up and seeing that the fee is paid but not confirmed, the client would need to get the coin and check the confirmations, and the wallet doesn't do that yet. Wasn't sure exactly how to move forward, but thanks for pointing this out.

Reply:
I've got most of a solution in ae6d26a, but I'm stuck for now because the fee payment recovery is triggered by signing in to the Decred wallet, and right now there's no way to sign in to the Decred wallet outside of the registration process. I'm already working on that in a separate PR though.